### PR TITLE
Update Glean dependency for mozilla_vpn

### PR DESCRIPTION
**Pull request by brizental**
Ok, so. I thought it would be fine to just keep glean.js in there as a dependency until we validate the glean-core data and yank glean.js out (https://github.com/mozilla/probe-scraper/pull/556#event-9352957919).

Turns out that was a bad assesment, because now we are sending data with glean-core using the `metrics` ping, but there is no `mozilla_vpn.metrics` table generated.

If I am not mistaken -- calling on @badboy and @travis79 for a double check on this -- glean-js is just a subset of glean-core. So if we have both, we can just have glean-core and there will be no problems anymore. Is that correct?

### CODE REVIEW

**Review by relud — COMMENTED** [review]
cc @sean-rose

**Comment by badboy** [comment]
> Ok, so. I thought it would be fine to just keep glean.js in there as a dependency until we validate the glean-core data and yank glean.js out ([#556 (comment)](https://github.com/mozilla/probe-scraper/pull/556#event-9352957919)).
> 
> Turns out that was a bad assesment, because now we are sending data with glean-core using the `metrics` ping, but there is no `mozilla_vpn.metrics` table generated.
> 
> If I am not mistaken -- calling on @badboy and @travis79 for a double check on this -- glean-js is just a subset of glean-core. So if we have both, we can just have glean-core and there will be no problems anymore. Is that correct?

That's correct. This should work out and a dry-run runs through on this.

**Review by sean-rose — APPROVED** [review]
LGTM

**Comment by sean-rose** [comment]
@brizental this is blocked waiting for a code owner to approve, and is a blocker for [DENG-996](https://mozilla-hub.atlassian.net/browse/DENG-996) "Recover mozilla_vpn metrics ping data".

**Review by relud — APPROVED** [review]

**Pull request merged**
